recipes/allantools/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "allantools" %}
+{% set version = "2019.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/be/9d/f36781c34001f4e53d8e6a368829c82937cee77c419e09caf7badc8d6e9d/AllanTools-2019.7.tar.gz
+  sha256: cd715068538ac1e457d88251fa166e103c536a861edae4458e970b9d4edf316b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - setuptools
+    - pytest-runner
+    - numpydoc
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - matplotlib
+    - pytest
```

Comment:
Upstream only seems to require numpy and scipy: https://github.com/aewallin/allantools/blob/32c9055a4811899fde6ae2013dbf40e8aedf7ab6/setup.py#L23